toolchains.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF8"?>
+<toolchains>
+    <!-- JDK toolchains -->
+    <toolchain>
+        <type>jdk</type>
+        <provides>
+            <version>15</version>
+            <vendor>openJDK</vendor>
+        </provides>
+        <configuration>
+            <jdkHome>/usr/lib/jvm/java-15-oracle</jdkHome>
```

Comment:
This is a good point to include a comment like `<jdkHome><your path here />/jdk-15<jdkHome>`

Reply:
Done

Reply:
Also changed the file location